### PR TITLE
Add GPU recommendation JSON in Docs

### PR DESCRIPTION
**Pull request by bharathappali**
## Description

Adds the sample JSON response which has GPU recommendations

### Type of change

- [ ] Bug fix
- [ ] New feature
- [x] Docs update
- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
- [ ] Requires DB changes

## How has this been tested?

No tests needed as it's a doc change

- [ ] New Test X
- [ ] Functional testsuite

## Checklist :dart:

- [x] Followed coding guidelines
- [x] Comments added
- [x] Dependent changes merged
- [x] Documentation updated
- [x] Tests added or updated

## Additional information

None

### CODE REVIEW

File: design/MonitoringModeAPI.md [hunk 1]
```diff
@@ -2960,6 +2960,361 @@ Returns the recommendation at a particular timestamp if it exists
...
+                  "message": "Recommendations Are Available",
+                  "code": 111000
+                }
+              },
+              "data": {
+                "2024-08-09T10:07:09.000Z": {
+                  "notifications": {
+                    "111104": {
+                      "type": "info",
+                      "message": "Fixed Term Recommendations Available",
```
Comment (chandrams) [inline design/MonitoringModeAPI.md:3000]: Is this fixed term recommendations always?

File: design/MonitoringModeAPI.md [hunk 2]
```diff
@@ -2960,6 +2960,361 @@ Returns the recommendation at a particular timestamp if it exists
...
+                        "performance": {
+                          "pods_count": 1,
+                          "confidence_level": 0.0,
+                          "config": {
+                            "limits": {
+                              "memory": {
+                                "amount": 1.1090952192E9,
+                                "format": "bytes"
+                              },
+                              "nvidia.com/mig-4g.20gb": {
```
Comment (chandrams) [inline design/MonitoringModeAPI.md:3075]: Will cpu also be available along with GPU recommendations?

File: design/MonitoringModeAPI.md [hunk 3]
```diff
@@ -2960,6 +2960,361 @@ Returns the recommendation at a particular timestamp if it exists
...
+                            "memoryUsage": {
+                              "min": 0.0,
+                              "q1": 0.0,
+                              "median": 0.0,
+                              "q3": 0.0,
+                              "max": 3.058688E8,
+                              "format": "bytes"
+                            }
+                          },
+                          "2024-08-02T10:07:09.000Z": {
```
Comment (chandrams) [inline design/MonitoringModeAPI.md:3154]: Format is missing here? Is box plots applicable for GPU, then we should have GPU usage also isn't it?

**Comment by bharathappali** [comment]
PR #1325 will be adding the correct changes, closing this PR as it's no longer needed.

**Pull request closed without merging**